src/t_stream.c

```diff
@@ -3083,7 +3087,7 @@ void xautoclaimCommand(client *c) {
     }
 
     /* Do the actual claiming. */
-    streamConsumer *consumer = NULL;
+    streamConsumer *consumer = streamLookupConsumer(group,c->argv[3]->ptr,1);
```

Comment:
same here.. the refresh used to be inside the `count` loop.
if this is a bug fix or behavior change, we mention it in the PR top comment.

Reply:
done